content/en/integrations/guide/source-code-integration.md

```diff
@@ -245,7 +259,7 @@ You can directly access a trace in its source repository on GitHub in the [Conti
 
 ### Inline source code
 
-If you are a GitHub SaaS user, install Datadog's [GitHub integration][2] to directly inline code snippets from your GitHub repository in your stack traces in [Error Tracking][8]. When specifying permissions on the integration tile, enable Datadog read permissions to **Contents**.
+If you are a GitHub or GitLab SaaS user, configure your [provider integration](#configure-your-repositories) to directly inline code snippets from your repository in your stack traces in [Error Tracking][8].
```

Comment:
I find it a bit confusing:
- in "GitHub or GitLab SaaS user", it's not clear whether "SaaS" refers to GitLab only or GItHub as well
- I'm not sure "GitLab SaaS" is very commonly used, maybe "gitlab.com" is more explicit ?
- for GitLab, there is no provider integration to configure and it's coming out of the box
GitHub or GitLab SaaS userHow about updating this section to have two separate tabs for GitHub and GitLab ?